src/pypgstac/Cargo.toml

```diff
@@ -1,17 +1,14 @@
 [package]
 name = "pgstacrs"
-version = "0.7.10-dev"
+version = "0.0.0"
 edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [lib]
 path = "src/lib.rs"
-name = "pgstacrs"
+name = "pypgstac"
```

Comment:
I think this is the name of the python package generated by the maturin. I'd recommend for it to be something like `_rust` or `_pgstacrs` so that 1) it's clear it's internal code not expected to be called by the user (re-exporting anything that we need directly through python code) and 2) making it clear when reading the code when a function is imported from the generated rust.

As it is, `from pypgstac.pypgstac import *` would I think import Rust functions, but it's easy to forget that when reading

Reply:
Also since the `#[pymodule]` is still set to `pgstacrs`, not sure if this builds correctly?

Reply:
Yeah, you'd have to change the name over there too, and probably in the python code.

Reply:
It unexpectedly seems to still build for me... I can't tell if it's actually working or if maybe I'm loading cached artifacts